src/Dockerfile

```diff
@@ -0,0 +1,33 @@
+#See https://aka.ms/containerfastmode to understand how Visual Studio uses this Dockerfile to build your images for faster debugging.
+
+FROM mcr.microsoft.com/dotnet/aspnet:5.0 AS base
+WORKDIR /app
+EXPOSE 80
+EXPOSE 443
+
+#Build angular ionic
```

Comment:
Here is the angular ionic build process in the multi stage docker file [Use multi-stage builds](https://docs.docker.com/develop/develop-images/multistage-build/)